d2renderers/d2svg/d2svg.go

```diff
@@ -83,13 +85,18 @@ type RenderOpts struct {
 	MasterID string
 }
 
-func dimensions(diagram *d2target.Diagram, pad int) (left, top, width, height int) {
+func dimensions(diagram *d2target.Diagram, pad int, hasShadow bool) (left, top, width, height int) {
 	tl, br := diagram.BoundingBox()
 	left = tl.X - pad
 	top = tl.Y - pad
 	width = br.X - tl.X + pad*2
 	height = br.Y - tl.Y + pad*2
 
+	if hasShadow {
```

Comment:
we should adjust the specific shape's bounds instead of adding it here since it may not influence the diagram bounding box if it is not on one of the outside borders

I think we can add another check similar to this one, probably as the first check due to the smaller values:

https://github.com/terrastruct/d2/blob/36a8aa679a5091a23207d8f9a5c151d58f1bc7d7/d2target/d2target.go#L169-L173

Reply:
That makes sense, I'll make the changes.
Thanks!